readme.md

```diff
@@ -17,17 +17,18 @@ The module can be downloaded from the [releases page](https://github.com/hms-net
 * Realtime Read functionality
    * Faster tag value update rates via the Talk2M API
    * Supports up to once per second update rates
-   * Configured on a tag by tag basis
+   * Configured on a tag by tag basis by setting a tag's `Realtime` property to true
+   * Configured on a per Ewon basis by setting the `AllRealtime` tag in an Ewon's `_config` directory to true.
 * Write functionality
    * Allows Ignition to modify tag values on a Flexy
 
 ## Installation
 
-The module can be installed by navigating to the Ignition web server.  Once on the webserver go to the "Config" section then under the "System" category click the "Modules" page.  At the bottom of the "Modules" page click "Install or Upgrade a Module...".  Follow the prompts to install the module.
+The module can be installed by navigating to the Ignition web server.  Once on the webserver go to the `Config` section then under the `System` category click the `Modules` page.  At the bottom of the `Modules` page click `Install or Upgrade a Module...`.  Follow the prompts to install the module.
```

Comment:
At some point, not necessarily this PR, the readme needs some grammar work. The installation section and below read very poorly.

Reply:
Im thinking of just ditching the readme related commits and rewriting the readme tomorrow.  I think when I wrote it originally I was in a rush to get something together and never looked at it since